apps/nova/src/components/TutorProfile/Rating.tsx

```diff
@@ -46,22 +49,38 @@ const Rating: React.FC<{ rating: IRating.Populated }> = ({ rating }) => {
               <p>{dayjs(rating.createdAt).fromNow()}</p>
             </p>
           </div>
-          <div className="flex gap-3">
-            <Button onClick={showEditDialog} size={ButtonSize.Tiny} type={ButtonType.Primary}>
-              <Edit className="w-4 text-white" />
-            </Button>
-            <Button onClick={showDeleteDialog} size={ButtonSize.Tiny} type={ButtonType.Primary}>
-              <Trash2 className="w-4 text-white" />
-            </Button>
-          </div>
+          {profile?.id === rating.rater.id && (
+            <div className="flex gap-3">
+              <Button
+                onClick={showEditDialog}
+                size={ButtonSize.Tiny}
+                type={ButtonType.Primary}
+              >
+                <Edit className="w-4 text-white" />
+              </Button>
+              <Button
+                onClick={showDeleteDialog}
+                size={ButtonSize.Tiny}
+                type={ButtonType.Primary}
+              >
+                <Trash2 className="w-4 text-white" />
+              </Button>
+            </div>
+          )}
         </div>
         <p>{rating.feedback}</p>
-        <EditRating
-          open={edit.open}
-          close={edit.hide}
-          rate={rating}
-          title={intl("tutor.rate.editmessage")}
-        />
+        {profile?.id === rating.rater.id && (
+          <>
+            <EditRating open={edit.open} close={edit.hide} rate={rating} />
+
+            <DeleteRating
+              tutorId={rating.ratee.id}
+              open={deleteRating.open}
+              close={deleteRating.hide}
+              id={rating.id}
+            />
+          </>
+        )}
```

Comment:
Let's use the ternary operator syntax when conditionally render components. Not to accidentally render something unexpected. 

```suggestion
        {profile?.id === rating.rater.id ? (
          <>
            <EditRating open={edit.open} close={edit.hide} rate={rating} />

            <DeleteRating
              tutorId={rating.ratee.id}
              open={deleteRating.open}
              close={deleteRating.hide}
              id={rating.id}
            />
          </>
        ) : null}
```